studio/lib/constants/infrastructure.ts

```diff
@@ -43,9 +43,13 @@ export const PRICING_TIER_FREE_KEY = 'FREE'
 export const PRICING_TIER_DEFAULT_KEY = 'FREE'
 
 export const PROVIDERS = {
+  FLY: {
+    id: 'FLY',
+    name: 'Fly.io',
+  },
   AWS: {
     id: 'AWS',
-    name: 'aws',
+    name: 'AWS',
     DEFAULT_SSH_KEY: 'supabase-app-instance',
     regions: {
       EAST_US: 'us-east-1', // North Virginia
```

Comment:
these regions seemed unused - the [slug] code probably needs to be updated to use these rather than the 'REGIONS' field that's currently used

Reply:
think your right. just checking this now, see if we can clean it up